### DISCUSSION TITLE
캐릭터 움직임 질문

### DISCUSSION THREAD

**Discussion by jun5640** [naverz/zepeto-studio-kor#144] [5. Q&A]
현재 월드로 개발을 진행중입니다.

캐릭터가 생성된 후에 컨트롤 패드를 이용하여 움직여서 약간 천천히 뛰기를 하면 
Idle -> Walk -> Run 순으로 애니메이션이 바뀌는데 Walk -> Run으로 바뀌는 과정이 너무 길어 캐릭터가 미끄러지는 것처럼 보입니다.
수정을 할려고 해도 생성되는 캐릭터가 Zepeto Players Script에 넣은 Animator를 사용하지 않는 것 같던대 어떻게 해야하나요?

**Comment by moondory77-naverz** [comment]
안녕하세요. 
해당 스레드 참고 부탁드리겠습니다.

[제페토 캐릭터 애니메이션 수정](https://github.com/naverz/zepeto-script-sample/issues/65)

참고로 제페토 캐릭터 컨트롤러 모듈을 통해 생성되는 캐릭터는
ZepetoPlayers 컴포넌트의 Player >> AnimatorController에 지정된 animator를 instantiate 하여 사용되며, 
런타임에 ZepetoCharacter 컴포넌트에 의해, 스크립트로 제어되고 있습니다.
